src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/entities/AsylumCaseFieldDefinition.java

```diff
@@ -763,6 +764,41 @@
     ADJOURN_HEARING_WITHOUT_DATE_REASONS(
             "adjournHearingWithoutDateReasons", new TypeReference<String>(){}),
 
+    IS_FEE_PAYMENT_ENABLED(
+            "isFeePaymentEnabled", new TypeReference<YesOrNo>() {}),
+    PAY_FOR_THE_APPEAL_OPTION(
+            "payForTheAppealOption", new TypeReference<String>() {}),
+    APPEAL_FEE_HEARING_DESC(
+            "appealFeeHearingDesc", new TypeReference<String>(){}),
+    APPEAL_FEE_WITHOUT_HEARING_DESC(
+            "appealFeeWithoutHearingDesc", new TypeReference<String>(){}),
+    FEE_HEARING_AMOUNT_FOR_DISPLAY(
+            "feeHearingAmountForDisplay", new TypeReference<String>(){}),
+    FEE_WITHOUT_HEARING_AMOUNT_FOR_DISPLAY(
+            "feeWithoutHearingAmountForDisplay", new TypeReference<String>(){}),
+    PAYMENT_STATUS(
+            "paymentStatus", new TypeReference<String>(){}),
+    FEE_PAYMENT_APPEAL_TYPE(
+            "feePaymentAppealType", new TypeReference<String>(){}),
+    PAYMENT_REFERENCE(
+            "paymentReference", new TypeReference<String>() {}),
+    FEE_CODE(
+            "feeCode", new TypeReference<String>(){}),
+    FEE_DESCRIPTION(
+            "feeDescription", new TypeReference<String>(){}),
+    FEE_VERSION(
+            "feeVersion", new TypeReference<String>(){}),
+    FEE_AMOUNT(
+            "feeAmount", new TypeReference<BigDecimal>(){}),
+    PBA_NUMBER(
+            "pbaNumber", new TypeReference<String>(){}),
+    PAYMENT_DESCRIPTION(
+            "paymentDescription", new TypeReference<String>(){}),
+    ERROR_CODE(
+            "errorCode", new TypeReference<String>(){}),
+    ERROR_MESSAGE(
```

Comment:
Same as above.

I just wonder if it's worth changing this field to PAYMENT_ERROR_MESSAGE (as ERROR_MESSAGE is quite generic). It might be prudent to change this now so it's specific to our payments feature. We should possibly run this past the other devs @makowalski ?

Reply:
Changed field to PAYMENT_ERROR_MESSAGE